vault/mount.go

```diff
@@ -795,8 +797,23 @@ func requiredMountTable() *MountTable {
 		Description: "system endpoints used for control, policy and debugging",
 		UUID:        sysUUID,
 	}
+
+	identityUUID, err := uuid.GenerateUUID()
+	if err != nil {
+		panic(fmt.Sprintf("could not create identity mount entry UUID: %v", err))
+	}
+
+	identityMount := &MountEntry{
+		Table:       mountTableType,
+		Path:        "identity/",
+		Type:        "identity",
+		Description: "identity store",
+		UUID:        identityUUID,
+	}
+
```

Comment:
Move the writing to `sync.Once`. But not adding invalidation because there is no storage entry that affects the state of identity store.